SECURITY.md

```diff
@@ -0,0 +1,68 @@
+# Security Policy
+As an open-source project, we understand the importance of and responsibility
+for security. This Security policy outlines our guidelines and procedures for
+ensuring the highest level of Security and trust for our users who consume the
+oneAPI Construction Kit.
+
+## Supported Versions
+We will perform patch releases for the supported [latest version][1] or create
+new releases which contain fixes for security vulnerabilities and important
+bugs.
+
+## Report a Vulnerability
+We are very grateful to the security researchers and users that report back
+security vulnerabilities. We investigate every report thoroughly.
+We strongly encourage you to report security vulnerabilities to us privately,
+before disclosing them on public forums or opening a public GitHub issue.
+Report a vulnerability to us in one of two ways:
+* Open a draft [**GitHub Security Advisory**][2]
+* Send e-mail to the following address: **security@uxlfoundation.org**.
+Along with the report, please include the following info:
+  * A descriptive title.
+  * Your name and affiliation (if any).
+  * A description of the technical details of the vulnerabilities.
+  * A minimal example of the vulnerability so we can reproduce your findings.
+  * An explanation of who can exploit this vulnerability, and what they gain
+  when doing so.
```

Comment:
Style wise, not flagged by markdownlint, personally for readability I prefer indentation of subsequent lines in a list:
```suggestion
  * An explanation of who can exploit this vulnerability, and what they gain
    when doing so.
```
Likewise for others below.